src/main/java/org/jabref/model/database/BibDatabaseContext.java

```diff
@@ -198,7 +198,13 @@ public boolean isBiblatexMode() {
 
         // 4. BIB file directory
         getDatabasePath().ifPresent(dbPath -> {
-            String parentDir = dbPath.getParent().toAbsolutePath().toString();
+            assert dbPath != null : "dbPath is null";
+            Path parentPath = dbPath.getParent();
+            if( parentPath == null ) {
```

Comment:
Fixed in e56e68c. Sorry, I've overlooked that (and Eclipse overlooked it as well ;)